### PR TITLE
Feature/migrate signing box to new dss

### PR DESCRIPTION
**Pull request by janvdbergh**

### CODE REVIEW

File: eid-dss-portal-control/src/test/java/TestJan.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import org.jboss.seam.core.Interpolator;
+
+public class TestJan {
```
Comment (hansvandenbroeck) [inline eid-dss-portal-control/src/test/java/TestJan.java]: TestJan?

File: eid-dss-portal-control/src/main/java/be/fedict/eid/dss/portal/servlet/SignatureResponseServlet.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+import be.e_contract.dssp.client.DigitalSignatureServiceSession;
+import be.e_contract.dssp.client.SignResponseVerifier;
+import be.e_contract.dssp.client.exception.ClientRuntimeException;
+import be.e_contract.dssp.client.exception.SubjectNotAuthorizedException;
+import be.e_contract.dssp.client.exception.UserCancelException;
+import be.fedict.eid.dss.portal.control.state.SigningModel;
+import be.fedict.eid.dss.portal.control.state.SigningModelRepository;
+
+public class SignatureResponseServlet extends HttpServlet {
```
Comment (hansvandenbroeck) [inline eid-dss-portal-control/src/main/java/be/fedict/eid/dss/portal/servlet/SignatureResponseServlet.java:25]: Is there a test for this?

File: eid-dss-portal-control/src/main/java/be/fedict/eid/dss/portal/control/state/SigningModelRepository.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package be.fedict.eid.dss.portal.control.state;
+
+import javax.servlet.http.HttpSession;
+
+public class SigningModelRepository {
```
Comment (hansvandenbroeck) [inline eid-dss-portal-control/src/main/java/be/fedict/eid/dss/portal/control/state/SigningModelRepository.java:5]: test?

**Review by hansvandenbroeck — CHANGES_REQUESTED** [review]
Is it possible I miss some tests?

File: eid-dss-portal-control/src/test/java/TestJan.java (hunk 1 above)
Comment (janvdbergh) [reply eid-dss-portal-control/src/test/java/TestJan.java]: Removed it.

File: eid-dss-portal-control/src/main/java/be/fedict/eid/dss/portal/servlet/SignatureResponseServlet.java (hunk 2 above)
Comment (janvdbergh) [reply eid-dss-portal-control/src/main/java/be/fedict/eid/dss/portal/servlet/SignatureResponseServlet.java:25]: Codebase does not have tests :)

File: eid-dss-portal-control/src/main/java/be/fedict/eid/dss/portal/control/state/SigningModelRepository.java (hunk 3 above)
Comment (janvdbergh) [reply eid-dss-portal-control/src/main/java/be/fedict/eid/dss/portal/control/state/SigningModelRepository.java:5]: Codebase does not have tests :)

**Pull request merged**